Dockerfile

```diff
@@ -1,6 +1,17 @@
 # Build the manager binary
 FROM golang:1.17 as builder
 
+RUN apt-get update && apt-get install -y ca-certificates openssl
```

Comment:
Is there a reason for the extra certificate installs?

Reply:
Removed in the latest commit. Let's validate with the integration tests if this is needed or not. 

I had a local issue in which I needed those. Also during my testing, I noted some int-tests failures because these. I will update once the tests finish.